container-registry/container-analysis/pom.xml

```diff
@@ -27,34 +27,41 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
   </properties>
 
+  <!-- [START dependencies] -->
+  <dependencyManagement>
```

Comment:
Should we consider adding a comment explaining what it is or why we commend it? May confuse users not familiar with the concept. 

Do we have an existing doc or something that we can link to?

Reply:
It's talked about in the CONTRIBUTING guide as a change from yesterday, but the same note here wouldn't go amiss.

Reply:
SG, I can add a link to the libraries-bom page in the comments